tools/src/main/python/dlpx/virtualization/_internal/cli.py

```diff
@@ -32,6 +32,12 @@
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'],
                         obj=click_util.ConfigFileProcessor.read_config())
 
+# This setting is added to workaround the bug in click 7.1 on windows
+# when case_sensitive=False is used on click.Options
```

Comment:
for multi line comments add a # the line before and after as per the python style guide